<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProductos</class>
 <widget class="QMainWindow" name="frmProductos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestión de miEmpresa - Productos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gestión de Productos</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>541</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Producto</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>31</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Código:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>63</y>
       <width>99</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Precio de compra:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCodigo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:5px;</string>
     </property>
     <property name="placeholderText">
      <string>Código del producto</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtObservaciones">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>511</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:10px;</string>
     </property>
     <property name="placeholderText">
      <string>Escriba su comentario</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Observaciones:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>94</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Precio de venta:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Tipo de IVA:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboTipoIVA">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Sin IVA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21% : IVA General</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10% : IVA Reducido</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4% : IVA Superreducido</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Otro</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="chbDescatalogado">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Aplica algún tipo de recargo</string>
     </property>
     <property name="text">
      <string>Descatalogado</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxPrecio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:5px;</string>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxPVP">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:5px;</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>29</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:5px;</string>
     </property>
     <property name="placeholderText">
      <string>Ingrese nombre del producto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Stock</string>
    </property>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>35</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Stock:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>77</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Stock mínimo:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxStock">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:5px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxStockMinimo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#d1ccc0;
border-radius:5px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#0fb9b1;
color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#F00;
color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#0fb9b1;
color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#0fb9b1;
color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtCodigo</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>spinBoxPrecio</tabstop>
  <tabstop>spinBoxPVP</tabstop>
  <tabstop>comboTipoIVA</tabstop>
  <tabstop>chbDescatalogado</tabstop>
  <tabstop>txtObservaciones</tabstop>
  <tabstop>spinBoxStock</tabstop>
  <tabstop>spinBoxStockMinimo</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnEliminar</tabstop>
  <tabstop>btnLimpiar</tabstop>
  <tabstop>btnSalir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
